### PR TITLE
[#3] [iOS] [UI] As a user, I can see the splash screen

### LINKED ISSUE #3: [iOS] [UI] As a user, I can see the splash screen
## Why

When users open the app, the mobile app should allow users to see the splash screen with the Nimble logo.

## Acceptance Criteria

- Display the status bar in white color.
- Display the image background with an overlay.
- Display the Nimble logo.
- Implement animation to show the logo based on Figma design.

## Design

Image

Link: https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=31%3A0

### PR DESCRIPTION
**Pull request by suho**
Resolve #3

## What happened 👀

- Create the splash screen for the iOS application

## Insight 📝

- Edited the `LaunchScreen.storyboard` file by following the design

## Proof Of Work 📹

Screen Shot 2022-08-30 at 14 48 37

### CODE REVIEW

**Review by edgarss — APPROVED** [review]

File: ios/Surveys/Resources/Assets/Assets.xcassets/logoWhite.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    {
+      "filename" : "Logo White.pdf",
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```
Comment (nkhanh44) [inline ios/Surveys/Resources/Assets/Assets.xcassets/logoWhite.imageset/Contents.json:12]: should we enable preserve vector data for this PDF? if so we won't need care about scaling images when their size changed 😄 .
Same for all PDFs below.

**Review by nkhanh44 — APPROVED** [review]
LGTM, just one optional suggestion 👍 .

File: ios/Surveys/Resources/Assets/Assets.xcassets/logoWhite.imageset/Contents.json (hunk 1 above)
Comment (suho) [reply ios/Surveys/Resources/Assets/Assets.xcassets/logoWhite.imageset/Contents.json:12]: Good catch!

Comment (suho) [reply ios/Surveys/Resources/Assets/Assets.xcassets/logoWhite.imageset/Contents.json:15]: Fixed here: https://github.com/suho/kmm-ic/pull/78/commits/37bd5a9a2e6a12f1d1fa04ec84d001038ab23f88

**Pull request merged**